### DISCUSSION TITLE
Pytest order with parallel test execution

### DISCUSSION THREAD

**Discussion by albertino87** [pytest-dev/pytest-order#108] [Ideas]
Hello,
to maximize the running efficiency of the tests running in parallel it is important that the slower tests run first, and i do it by marking the tests with the decorator @pytest.mark.order(0) from the pytest-order library. however this is not respected by xdist, can you possibly fix it?

**Comment by mrbean-bremen** [comment]
Did you check [the documentation](https://pytest-order.readthedocs.io/en/latest/other_plugins.html#usage-with-pytest-xdist)?

**Comment by albertino87** [comment]
yes I did, but that doesn't really solve the problem. I don't have a dependency problem. I don't care that all the test in one file run in the same thread. I only want that two very slow tests (in two different modules) run first. i don't see how that solves my problem
Many thanks

**Reply by mrbean-bremen** [reply]
You can only make sure that the slow tests inside the same module run first (if you are using the `--dist=loadfile` option). The order in which the `xdist` threads is out of control of this plugin.
You may check [this xdist issue]() and the referenced [pytest-slowest-first plugin](https://github.com/klimkin/pytest-slowest-first), this sounds as if it could help you.
